src/components/extensionTopContent/top-contents/GlobalNavigationBar.jsx

```diff
@@ -4,25 +4,39 @@ import { useContext } from "react";
 
 import ExtensionContext from "../../../context/ExtensionContext";
 
-function GnbButton({ iconShapeType }) {
+function GnbButton({ iconShapeType, onClick }) {
   return (
     <button className="w-8 text-lg">
-      <FontAwesomeIcon icon={iconShapeType} />
+      <FontAwesomeIcon
+        icon={iconShapeType}
+        onClick={onClick}
+      />
     </button>
   );
 }
 
 export default function GlobalNavigationBar() {
   const { bookmarkList } = useContext(ExtensionContext);
 
+  const hendleOnClickOpenOptionPage = () => {
+    if (chrome.runtime.openOptionsPage) {
+      chrome.runtime.openOptionsPage();
+    } else {
+      window.open(chrome.runtime.getURL("options.html"));
+    }
+  };
```

Comment:
```suggestion
  const hendleOnClickOpenOptionPage = () => {
    if (chrome.runtime.openOptionsPage) {
      chrome.runtime.openOptionsPage();
    }
  };
```
옵션 페이지를 여는 함수로 공식문서에서 찾기 힘들었는데 찾아주셔서 감사합니다. 다만, 위 코드 else의 존재여부가 궁금해져서 실제로 테스트를 해봤는데 else문이 있을 때와 없을 때 다른 동작을 찾기 어려웠습니다. 그래서 else 문의 역할과 어떤 부분에서 필요한지가 궁금합니다.

Reply:
저 또한 else 문이 없어도 동작하는 점을 확인하였으나 공식문서에서 else문에 대한 부분 기술해주었던 것을  이해를 못하여 코드를 그대로 차용하게 되었습니다. 다시 공식문서의 내용을 숙지하니 extention 내에 popup 창에서 호출하는 구문임을 확인하였습니다. 참조 주신 부분 참조하여 수정 하겠습니다.